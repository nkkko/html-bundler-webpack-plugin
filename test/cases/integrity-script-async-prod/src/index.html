<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <script async src="./main.js"></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>