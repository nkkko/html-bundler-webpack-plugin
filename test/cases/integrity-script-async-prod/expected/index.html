<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <script async src="main.js" integrity="sha384-E4IoJ3Xutt/6tUVDjvtPwDTTlCfU5oG199UoqWShFCNx6mb4tdpcPLu7sLzNc8Pe" crossorigin="anonymous"></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>